xmtp_mls/src/storage/encrypted_store/mod.rs

```diff
@@ -184,15 +185,21 @@ impl EncryptedMessageStore {
                 conn.raw_query(|conn| {
                     PoolTransactionManager::<AnsiTransactionManager>::commit_transaction(&mut *conn)
                 })?;
+                log::info!("Transaction being committed");
                 Ok(value)
             }
-            Err(err) => match conn.raw_query(|conn| {
-                PoolTransactionManager::<AnsiTransactionManager>::rollback_transaction(&mut *conn)
-            }) {
-                Ok(()) => Err(err),
-                Err(Error::BrokenTransactionManager) => Err(err),
-                Err(rollback) => Err(rollback.into()),
-            },
+            Err(err) => {
+                log::info!("Transaction being rolled back");
```

Comment:
We can probably switch some of these to debug logs when we're ready

Reply:
updated some here, will look a bit for other candidates a40d4459b670fa90175e886b564e811117e32633